napari/layers/surface/_surface_key_bindings.py

```diff
@@ -22,7 +22,7 @@ def activate_surface_transform_mode(layer):
 
 @register_surface_mode_action(trans._('Pan/zoom'))
 def activate_surface_pan_zoom_mode(layer: Surface):
-    layer.mode = Mode.PAN_ZOOM
+    layer.mode = str(Mode.PAN_ZOOM)
```

Comment:
shouldn't this be fine with the enum?

Reply:
See https://github.com/napari/napari/issues/6005 for more details on this - ideally this should be fixed, but since it touches on many different layers types we decided to delay fixing to a follow up PR.

Reply:
Ah, of course, I should have known, I'm the one who made that mess of types xD